docker-compose-raft.yml

```diff
@@ -4,38 +4,41 @@ services:
         working_dir: /habitat
         volumes:
             - .habitat/procs:/habitat/procs
```

Comment:
is there a reason we don't copy over the whole `.habitat/` folder?

Reply:
voluming in the communities directory is usually unwanted IMO